### PR TITLE
docs: mismatch between explanation and code in triggers

### LINKED ISSUE #33777: Mismatch between explanation and code about triggers used
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

Publishing Docker images (https://docs.github.com/en/actions/publishing-packages/publishing-docker-images) 

### What part(s) of the article would you like to see updated?

1) In the section "[Publishing images to Docker Hub](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub)" it says:

> The workflow in the example below runs when the release event triggers with the **created** activity type.

while the code uses the **published** activity type:

```
on:
  release:
    types: [published]
```

2) In the section [Publishing images to GitHub Packages](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages) is says:

> The workflow in the example below runs when the release event triggers with the created activity type. 

while the code uses a push trigger:

```
on:
  push:
    branches: ['release']

```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
### Why: To fix mismatch between explanation and code in triggers

Closes: #33777

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@gagan-bhullar-tech Hello! 👋 Thank you for opening a PR! 

We will need to put this PR on hold for now until the issue this PR is linked to has been reviewed by the team 💛

File: content/actions/publishing-packages/publishing-docker-images.md
Comment (hubwriter) [inline content/actions/publishing-packages/publishing-docker-images.md:55]: We could omit the link here:
```suggestion
Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when the `release` event triggers with the `published` activity type.
```

File: content/actions/publishing-packages/publishing-docker-images.md
Comment (hubwriter) [inline content/actions/publishing-packages/publishing-docker-images.md:132]: Slight correction here, and again I think we can omit the link.

```suggestion
Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when a change is pushed to the `release` branch.
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@gagan-bhullar-tech - Many thanks for raising this PR with these changes.

As a result of feedback we've had, we are currently making an effort to reduce the number of links between articles, except where necessary to the understanding of the current article, so I've suggested removing the links here.

If you'd like to click to commit my suggestions then I'll approve this and we can get this merged next week once the current repo freeze is lifted.

Thanks again for working on this.
👍

**Comment by gagan-bhullar-tech** [comment]
@hubwriter thanks for reviewing. Incorporated both changes.

**Comment by hubwriter** [comment]
@gagan-bhullar-tech - Many thanks. We'll merge this next week after the current repo freeze is lifted.
🚀

**Review by hubwriter — APPROVED** [review]

**Pull request merged**
